### PR TITLE
fix(ui): put investor metric into a mixin

### PR DESCRIPTION
**Pull request by angelicawill**
(resolves: #543)

### CODE REVIEW

**Comment by angelicawill** [comment]
@zolotokrylin merge conflicts are resolved.

**Comment by zolotokrylin** [comment]
@angelicawill you can reuse this PR (just rename it) for this issue #543

**Comment by zolotokrylin** [comment]
Looks like it is a duplicate

File: src/lib/components/Portfolio/template.pug [hunk 1]
```diff
@@ -36,7 +47,7 @@ mixin metric(type, value=null)
     when 'investors'
       +metricContainer
         +metricTitle("Investors")
-        block
+        +metricValueInvestors
```
[the remark this answers is not in the document]
Comment (zolotokrylin) [reply src/lib/components/Portfolio/template.pug:50]: Any errors in console?

**Comment by zolotokrylin** [comment]
@angelicawill please don't use capital letters in naming, unless it is a term that needs to be highlighted. Similar to what we have in programming. 

@markcurchin @georgeciubotaru @Addis-Belete FYI

**Pull request merged**